Managers/SessionManager.cs

```diff
@@ -0,0 +1,90 @@
+using CoronaCheckIn.Models;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+namespace CoronaCheckIn.Managers
+{
+    public class SessionManager
+    {
+        private ApplicationDbContext Context { get; set; }
+
+        public SessionManager(ApplicationDbContext context)
+        {
+            Context = context;
+        }
+
+        public IEnumerable<Session> GetSessions(Room? room = null, Guid? roomId = null, User? user = null, string? userId = null, bool? isInfected = null, DateTime? after = null, DateTime? before = null)
+        {
+            var queryable = Context.Sessions.AsQueryable();
+
+            if (room != null)
+            {
+                queryable = queryable.Where(session => session.RoomId == room.Id);
+            }
+            if (roomId != null)
+            {
+                queryable = queryable.Where(session => session.RoomId == roomId);
+            }
+            if (user != null)
+            {
+                queryable = queryable.Where(session => session.UserId == user.Id);
+            }
+            if (userId != null)
+            {
+                queryable = queryable.Where(session => session.UserId == userId);
```

Comment:
userID muss in einen String umgewandelt werden

Reply:
ist doch schon nen string?

Reply:
ohh fuck sorry 😅